src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java

```diff
@@ -393,6 +383,28 @@ else if (answer1 instanceof ShortAnswerSubmittedAnswer submittedAnswer1 && answe
         return true;
     }
 
+    /**
+     * Returns {@code true} if the quiz submissions are equal to each other
+     * and {@code false} otherwise.
+     *
+     * @param answer1 a quiz submission
+     * @param answer2 a quiz submission to be compared with {@code submission1} for equality
+     * @return {@code true} if the quiz submissions are equal to each other and {@code false} otherwise
+     * @throws RuntimeException if the answer types are not supported
+     */
+    public static boolean isContentEqualTo(SubmittedAnswer answer1, SubmittedAnswer answer2) {
+        if (answer1 instanceof DragAndDropSubmittedAnswer submittedAnswer1 && answer2 instanceof DragAndDropSubmittedAnswer submittedAnswer2) {
+            return isContentEqualTo(submittedAnswer1, submittedAnswer2);
+        }
+        else if (answer1 instanceof MultipleChoiceSubmittedAnswer submittedAnswer1 && answer2 instanceof MultipleChoiceSubmittedAnswer submittedAnswer2) {
+            return isContentEqualTo(submittedAnswer1, submittedAnswer2);
+        }
+        else if (answer1 instanceof ShortAnswerSubmittedAnswer submittedAnswer1 && answer2 instanceof ShortAnswerSubmittedAnswer submittedAnswer2) {
+            return isContentEqualTo(submittedAnswer1, submittedAnswer2);
+        }
+        throw new RuntimeException("Not supported");
```

Comment:
Directly add the log message here instead of throwing and catching an exception.

Also use a static variable for the logger instead of calling `getLogger` repeatedly.